test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -118,11 +148,92 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<target name="TCK.move.dump.ddrext" description="Move core file from MVS to HFS">
 			<echo>Moving core file from MVS [${dump.name}${zos.core.suffix}] to HFS [${system.dump}]</echo>
 			<exec executable="/bin/mv" failonerror="true">
-				<arg line="&quot;//${dump.name}${zos.core.suffix}&quot; ${system.dump}"/>
+				<arg value="//${dump.name}${zos.core.suffix}"/>
+				<arg value="${system.dump}"/>
 			</exec>
 	</target>
 
 	<target name="clean" depends="TCK.run.tests.ddrext" description="clean">
 		<delete file="${system.dump}" />
 	</target>
+
+	<target name="ddrext.setup" description="Create the required directories">
+		<!-- do not invoke this target with antcall as the properties defined here need to be visible to all subsequent targets -->
+		<echo>Setting up TCK</echo>
+		<mkdir dir="${tck.output.dir}"/>
+		<mkdir dir="${tck.output.dir}_${tck.type}"/>
+		<echo>Recording Java version</echo>
+		<java fork="true" jvm="${sdk.executable}${sdk.executable.ext}" classname="com.ibm.dtfj.tck.harness.TCKLauncher" 
+			failonerror="true" outputproperty="jvm.version" newenvironment="true">
+			<jvmarg value="-version" />
+		</java>
+		<condition property="java.is64bit">
+			<matches pattern=" jvm\S+64| p\S+64|-64 " string="${jvm.version}" multiline="true"/>
+		</condition>
```

Comment:
I think this would be more reliable using `java -XshowSettings:properties -version` looking for `sun.arch.data.model = 64`.

Reply:
I changed`<property ... location="..."/>` and `sun.arch.data.model = 64` part. Please review again @keithc-ca